lib/preprocessor.cpp

```diff
@@ -721,6 +724,7 @@ void Preprocessor::reportOutput(const simplecpp::OutputList &outputList, bool sh
         case simplecpp::Output::INCLUDE_NESTED_TOO_DEEPLY:
         case simplecpp::Output::SYNTAX_ERROR:
         case simplecpp::Output::UNHANDLED_CHAR_ERROR:
+        case simplecpp::Output::EXPLICIT_INCLUDE_NOT_FOUND:
```

Comment:
can we skip this location for `simplecpp::Output::EXPLICIT_INCLUDE_NOT_FOUND` here?